modules/fts/pages/fts-architecture.adoc

```diff
@@ -0,0 +1,54 @@
+= FTS Architecture
+
+
+[abstract]
+
+
+The FTS Architecture is explained in the form of diagrams. Each diagram is co-located with its textual description.
+
+== Search Service Architecture
+
+The _Search_ Service provides extensive capabilities for natural-language querying.
+These include:
+
+* _Language-aware_ searching; allowing users to search for, say, the word `beauties`, and additionally obtain results for `beauty` and `beautiful`.
+* _Scoring_ of results, according to relevancy; allowing users to obtain result-sets that only contain documents awarded the highest scores.
+This keeps result-sets manageably small, even when the total number of documents returned is extremely large.
```

Comment:
Scoring just helps order the results.  The size of the result set is controlled independently.

Reply:
Update line 15 to "_Scoring_ determines the relevancy of the documents."